### PR TITLE
Pulsar Startup Script fixes to provide custom conf files for bookie s…

### PR DESCRIPTION
**Pull request by jai1**
## Motivation
There are two bugs in the start up script:-
a. While running `bin/bookkeeper` due to hardcoding in (bkenv.sh) we can't give a custom conf file with different zookeeper ports.
b. While running `bin/pulsar configuration-store` the script overrides the custom conf file with default if `PULSAR_GLOBAL_ZK_CONF` is not provided.

## Changes
Fixed both `bin/pulsar` and `conf/bkenv.sh` to accept custom config files.

### CODE REVIEW

File: bin/pulsar [hunk 1]
```diff
@@ -221,7 +221,6 @@ fi
 
 if [ -z "$PULSAR_GLOBAL_ZK_CONF" ]; then
     PULSAR_GLOBAL_ZK_CONF=$DEFAULT_GLOBAL_ZK_CONF
-    PULSAR_CONFIGURATION_STORE_CONF=$DEFAULT_GLOBAL_ZK_CONF
```
Comment (merlimat) [inline bin/pulsar]: This seems to be unrelated with the PR

Comment (jai1) [reply bin/pulsar]: > While running `bin/pulsar configuration-store` the script overrides the custom conf file with default if `PULSAR_GLOBAL_ZK_CONF` is not provided.

This change fixes this ^^

For example, try running 
```
export PULSAR_CONFIGURATION_STORE_CONF=<custom path>
bin/pulsar configuration-store
```

The DEFAULT_GLOBAL_ZK_CONF will be picked up instead of your custom file.

**Comment by jai1** [comment]
rerun java8 tests
